static/dbview.config.php

```diff
@@ -155,7 +155,6 @@
 			"author-hidden" => ["author", "hidden"],
 			"author-updated" => ["author", "updated"],
 			"author-gsid" => ["author", "gsid"],
-			"author-uri-id" => ["author", "uri-id"],
```

Comment:
Is there a reason why you removed these fields?

Reply:
Yes, they were duplicated keys. It ended up not mattering when creating the actual views because these duplicates would resolve to a single field so this is just housekeeping.

Reply:
Looks like too much copy&paste :grimacing: